src/main/resources/wa-task-configuration-employment-et_englandwales.dmn

```diff
@@ -406,7 +405,12 @@ else null</text>
           <text>"title"</text>
         </outputEntry>
         <outputEntry id="LiteralExpression_00rkvd5">
-          <text>LATEST_REVIEW_REFERRAL_TASK_TITLE</text>
+          <text>if (LATEST_REVIEW_REFERRAL_TASK_TITLE != null) then
+  LATEST_REVIEW_REFERRAL_TASK_TITLE
+else if (taskAttributes != null and taskAttributes.name != null) then
+  taskAttributes.name
+else
+  null</text>
```

Comment:
Rather than null which can cause the task configuration issue, make a 'default' name like Review Referral